healsparse/healSparseMap.py

```diff
@@ -1313,6 +1313,58 @@ def n_valid(self):
         self._n_valid = n_valid
         return n_valid
 
+    def get_valid_pixels_per_covpix(self):
+        """
+        Get all single covpixel valid_pixels, one at a time.
```

Comment:
This makes it sound like you return on pixel at a time.  Maybe "Generator to get valid_pixels associate with each coverage pixel, yielded one coverage pixel at a time"

Then it would be good to show usage since this is a generator.

Reply:
Also I don't see a test for this

Reply:
One of my big questions was about naming this routine.

Reply:
Other ideas:
```
iter_valid_pixels
iter_valid_pixels_by_covpix
```